docs/integrations/persisting-store-data.md

```diff
@@ -783,17 +783,19 @@ interface BearState {
     getItem: (name) => {
       const str = localStorage.getItem(name);
       if (!str) return null;
-      const { state } = JSON.parse(str);
+      const { state, version } = JSON.parse(str);
       return {
         state: {
           ...state,
           transactions: new Map(state.transactions),
         },
+        version
```

Comment:
This makes `version: undefined` if version is undefined. We also need to use object spread here like `setItem`.

Reply:
I've pushed another commit switching this as you've suggested.